index.html

```diff
@@ -0,0 +1,464 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <script src="index.js" defer></script> 
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <a name="top"></a>
+    <header>
+
+      
+        <h1 id="logo">Aziz Mejri</h1>
+        <button type="button" title="menu" class="menu-bars" id="menuBars" onclick="toggleMobileMenu()">
+          <i class="fas fa-bars"></i>
+        </button>
+        <nav id="desktop-nav">
+          <ul>
+            <li><a class="nav-link" href="#projects">Portoflio</a></li>
+            <li><a class="nav-link" href="#about-me">About</a></li>
+            <li>
+              <a class="nav-link" href="#contact"
+                >Contact <i class="far fa-envelope"></i
+              ></a>
+            </li>
+          </ul>
+        </nav>
+      
+
+      
+      <nav id="menuMobile" class="menu-mobile">
+        <ul class="list-menu">
+          <li class="list-menu-item"><a href="#top" >Hello</a></li>
+          <li class="list-menu-item"><a href="#projects" >Portfolio</a></li>
+          <li class="list-menu-item"><a href="#about-me" >About</a></li>
+          <li class="list-menu-item"><a href="#contact" >Contact</a></li>
+        </ul>
+        <i id="close-button" class="fas fa-times fa-2x cross invisible" onclick="toggleMobileMenu()"></i>
+    </nav>
```

Comment:
- [ ] It is a good practice for a frontend developer to follow the Figma design given. Though your design is not bad, I think it is better to follow the given mobile menu design.
Check the image of the design below 👇 

![image](https://user-images.githubusercontent.com/66097371/147299604-7a2c587f-88d9-4735-9c35-832cab2f7c96.png)

please find the link [here](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471)

![image](https://user-images.githubusercontent.com/66097371/147299677-82e96104-936d-4a17-ad8f-5b01baf2406c.png)

Reply:
Thanks a lot for the review and for your kind feedback! I'll work on it right now.